frontend/actions/playlisted_actions.js

```diff
@@ -15,9 +15,12 @@ export const REMOVE_PLAYLISTED = "REMOVE_PLAYLISTED";
 // Thunk Action creators
 export const createNewPlaylisted = ( songId, playlistId ) => dispatch => {
     return postPlaylisted( songId, playlistId )
-        .then( (playlistId) => dispatch( displayPlaylist(playlistId) ),
-            err => (console.log(err)) 
-        )
+        .then( (respObj) => {
+            dispatch( fetchPlaylists() );
+            dispatch( displayPlaylist(respObj.playlistId) );
+            return respObj.playlistId;
+        }, err => (console.log(err)
+        ))
 }
 
 export const removePlaylisted = (playlistedId) => dispatch => {
```

Comment:
I agree and am open to suggestions. I have a model called Playlist and a model called Playlisted which is a joins table between Playlist and Song. Since it is an association, I thought to use an adjective since a Playlisted instance is neither a Playlist nor Song..... Seemed the best option at the time.

Reply:
Hmm okay a couple suggestions: songsInPlaylist, songList, trackListed.